generated/nidcpower/nidcpower/session.py

```diff
@@ -132,7 +132,7 @@ class _SessionBase(object):
     Note: This property is not supported by all devices. Refer to Supported Properties by Device topic.
 
     Tip:
-    This property can use repeated capabilities. If set or get directly on the
+    This property can use repeated capabilities for channels. If set or get directly on the
     nidcpower.Session object, then the set/get will use all repeated capabilities in the session.
     You can specify a subset of repeated capabilities using the Python index notation on an
     nidcpower.Session repeated capabilities container, and calling set/get value on the result.
```

Comment:
This text is pretty old and not great. Here's a stab at improving it to better reflect the state of the art in our Python APIs and to be clearer to customers.

```
This property can be set/get to specific channels within you nidcpower.Session object. Use Python index notation on the repeated capabilities container `channels` to specify a subset.
Example: `my_session.channels[ ... ].active_advanced_sequence_step`,
To set/get on all, you can call the property directly on the nidcpower.Session:
`my_session.active_advanced_sequence_step`.
```

Reply:
#1604

Reply:
1. @marcoskirsch, does the fact that you created an issue imply that you'd rather I not switch to using your text in this PR?
2. The current text is parameterized, so that the "...for channels" part, in the first line, can change with the particular capability type(s). Does your new text also allow for that? Specifically, if I were to replace "... to specific **channels**" with, say, "... to specific **channels, instruments and pins**", would that still be okay?
3. As for the example in the second line, I could replace it with the first item in the types list (allowing for cases where an attribute is valid for more than one type), since I can't use multiple types in the example. But I suppose I'd also have to pick an attribute that makes sense for that capability type. That's going to make it a bit harder to parameterize this text, as @sbethur [requested](https://github.com/ni/nimi-python/pull/1599#discussion_r630524419) earlier in this PR.

Reply:
1. own PR, that's why I created the issue to track
2. yes
3. 👍🏻